signer/src/main.rs

```diff
@@ -69,7 +69,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // Configure the binary's stdout/err output based on the provided output format.
     let pretty = matches!(args.output_format, Some(LogOutputFormat::Pretty));
-    signer::logging::setup_logging("", pretty);
+    signer::logging::setup_logging("info,signer=debug", pretty);
```

Comment:
If we do this here does it mean we will override whatever it's set from outside? eg, in docker-compose

Reply:
yeah, the environment variable takes precedence. 

https://github.com/stacks-network/sbtc/blob/845f6dd0b9dfed81afebdab04061bd6b1241cfe4/signer/src/logging.rs#L22-L37